src/puara_filesystem.cpp

```diff
@@ -181,29 +254,13 @@ void JSONSettings::read_config_json_internal(std::string& contents)
 
 void JSONSettings::read_settings_json()
 {
-  std::cout << "json: Mounting FS" << std::endl;
-  spiffs.mount_spiffs();
-
-  std::cout << "json: Opening settings json file" << std::endl;
-  FILE* f = fopen("/spiffs/settings.json", "r");
-  if(f == NULL)
-  {
-    std::cout << "json: Failed to open file" << std::endl;
-    return;
-  }
-
-  std::cout << "json: Reading json file" << std::endl;
-  std::ifstream in("/spiffs/settings.json");
-  std::string contents(
-      (std::istreambuf_iterator<char>(in)), std::istreambuf_iterator<char>());
-
+  std::string contents = fs->read_file("settings.json");
```

Comment:
before we were reading in /spiffs/settings.json, will this keep the previous firmware working if someone is updating their firmware and want to keep the filesystem ?